vault/activity_log_util_common.go

```diff
@@ -74,7 +74,7 @@ func (a *ActivityLog) computeCurrentMonthForBillingPeriodInternal(ctx context.Co
 		if len(monthlyComputation) > 1 {
 			a.logger.Warn("monthly in-memory activitylog computation returned multiple months of data", "months returned", len(byMonth))
 		}
-		if len(monthlyComputation) >= 0 {
+		if len(monthlyComputation) > 0 {
```

Comment:
Actually, should we handle 0 months by returning some sort of error?

Reply:
I was under the impression that a length of 0 here just meant that there is no data. The way that this gets returned in the response is:

```
{
  "request_id": "ba3699b1-8b83-79a1-1b7d-ebd3496c88d4",
  "lease_id": "",
  "renewable": false,
  "lease_duration": 0,
  "data": {
    "by_namespace": [],
    "end_time": "2022-10-31T23:59:59Z",
    "months": [],
    "start_time": "2022-10-01T00:00:00Z",
    "total": {
      "distinct_entities": 0,
      "entity_clients": 0,
      "non_entity_tokens": 0,
      "non_entity_clients": 0,
      "clients": 0
    }
  },
  "wrap_info": null,
  "warnings": null,
  "auth": null
}```

Reply:
Okay, thought through this a bit more and 0-length monthly computation makes sense. The multi-month 0-length computation is also handled later on, since we range over `byMonth`, so all good!